src/Commands/AppEventsListener.php

```diff
@@ -102,9 +102,11 @@ private function runIteration(Subscription $subscription)
                 $job = AppEventFactory::fromMessage($message);
             } catch (UnserializableProtoException $e) {
                 if (! $this->option('silent')) {
-                    $this->info('No implementation registered for message type: ' . $e->protoMessageType);
+                    Log::warning('Proto is not registered', [
+                        'proto' => $e->protoMessageType,
+                    ]);
+                    $this->warning('No implementation registered for message type: ' . $e->protoMessageType);
                 }
-                $handledMessages[] = $message;
```

Comment:
There's plenty of service that don't and won't get a handler for the message, meaning message will never get acked and will be retried many times. Don't think that's the aim here.

Reply:
Ah, you're right. The goal of this PR is to stop ack-ing the pubsub messages for the events that are registered in `handlers` array of `app-events.php` but does not have the relevant proto class in `mappings` array of the same file. But I just noticed that I need to change the AppEventFactory for that, too. Will add changes in a minute.

Reply:
Took a bit more change to make that happen and I fall into the trap of updating everything unrelated but necessary.